### ISSUE TITLE
Won't install new games or update existing installs.

### ISSUE THREAD

**Issue by happysmack** [Heroic-Games-Launcher/HeroicGamesLauncher#1154]
### Describe the bug

I updated to 2.2.6 amd64 ( even tried going back to 2.2.3 ) and tried to install "In Sound Mind".  I just get the spinning arrows and nothing happens.  The same happens when I try to update Rocket League. It says installing 0%.  Then just stops.

### Add logs

Launch Command: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary launch Sugar --wine '/home/desk/.steam/steam/steamapps/common/Proton - Experimental/proton' --wine-prefix '/home/desk/Games/PROTON/RocketLeague'
        System Info:
        
  Heroic Version: 2.2.6 Oden
  Legendary Version:  0.20.6 A Red Letter Day
  OS: Ubuntu KERNEL: 5.13.0-37-generic ARCH: x64
  CPU: AMD Ryzen 7 3700X 8-Core Processor @3.6 GOVERNOR: ondemand
  RAM: Total: 67.4 GB Available: 58.7 GB
  GRAPHICS: GPU0: Navi 10 [Radeon RX 5600 OEM/5600 XT / 5700/5700 XT] VRAM: 512MB DRIVER:  
  PROTOCOL: x11
  
        Game Settings: {
	"checkUpdatesInterval": 10,
	"enableUpdates": false,
	"addDesktopShortcuts": false,
	"addStartMenuShortcuts": false,
	"autoInstallDxvk": true,
	"autoInstallVkd3d": false,
	"checkForUpdatesOnStartup": true,
	"customWinePaths": [],
	"defaultInstallPath": "/home/desk/Games/Heroic",
	"defaultWinePrefix": "/home/desk/Games/Heroic/Prefixes",
	"language": "en",
	"maxWorkers": 0,
	"minimizeOnLaunch": false,
	"nvidiaPrime": false,
	"otherOptions": "",
	"showUnrealMarket": true,
	"showFps": false,
	"useGameMode": false,
	"userInfo": {
		"epicId": "8a5372b5f93f4daf9822c65d9fda0f27",
		"name": "desk"
	},
	"wineCrossoverBottle": "Heroic",
	"winePrefix": "/home/desk/Games/PROTON/RocketLeague",
	"wineVersion": {
		"bin": "'/home/desk/.steam/steam/steamapps/common/Proton - Experimental/proton'",
		"name": "Proton - Proton - Experimental"
	},
	"altLegendaryBin": "",
	"altGogdlBin": "",
	"audioFix": false,
	"darkTrayIcon": true,
	"discordRPC": true,
	"egsLinkedPath": "'/home/desk/Games/epic-games-store'",
	"enableEsync": false,
	"enableFsync": false,
	"exitToTray": true,
	"maxRecentGames": 5,
	"offlineMode": false,
	"showMangohud": false,
	"startInTray": false,
	"autoSyncSaves": false,
	"enableFSR": false,
	"maxSharpness": 2,
	"enableResizableBar": false,
	"savesPath": "",
	"targetExe": ""
}

        Game Log:
        [cli] INFO: Logging in...
[Core] INFO: Trying to re-use existing login session...
[cli] INFO: Checking for updates...
[cli] ERROR: Game is out of date, please update or launch with update check skipping!

        

### Steps to reproduce

1. Go to 'In Sound Mind'
2. Click on 'install'
3. States that it is installing and immediately goes back  to previous state.
4. Says that an update is available and the loop continues.

### Expected behavior

I expect to click update and the update to be installed.  This just states that it is installing 0% then stops as if I did nothing at all.

### System Information

- OS [Ubuntu 20.04.4 LTS]: 
- Heroic Version [2.2.6]:

**Comment by happysmack** [comment]
(15:13:45) INFO:    [Legendary]:       Location: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/legendary
(15:13:45) INFO:    [Gog]:             Location: /opt/Heroic/resources/app.asar.unpacked/build/bin/linux/gogdl
(15:13:46) INFO:    [Backend]:         
  Heroic Version: 2.2.6 Oden
  Legendary Version:  0.20.6 A Red Letter Day
  OS: Ubuntu KERNEL: 5.13.0-37-generic ARCH: x64
  CPU: AMD Ryzen 7 3700X 8-Core Processor @3.6 GOVERNOR: ondemand
  RAM: Total: 67.4 GB Available: 58.8 GB
  GRAPHICS: GPU0: Navi 10 [Radeon RX 5600 OEM/5600 XT / 5700/5700 XT] VRAM: 512MB DRIVER:  
  PROTOCOL: x11
  
(15:13:46) WARNING: [Backend]:         Protocol already registered.
(15:13:46) INFO:    [Frontend]:        Refreshing Library
(15:13:46) INFO:    [Legendary]:       Refreshing library...
(15:13:46) INFO:    [Legendary]:       Refreshing Epic Games...
(15:13:46) INFO:    [Legendary]:       Refreshing Epic Games...
(15:13:47) INFO:    [Legendary]:       usage: legendary [-h] [-v] [-y] [-V]
                 {auth,install,download,update,repair,uninstall,launch,list-games,list-installed,list-files,list-saves,download-saves,sync-saves,verify-game,import-game,egl-sync,status,cleanup}
                 ...
legendary: error: argument subparser_name: invalid choice: 'list' (choose from 'auth', 'install', 'download', 'update', 'repair', 'uninstall', 'launch', 'list-games', 'list-installed', 'list-files', 'list-saves', 'download-saves', 'sync-saves', 'verify-game', 'import-game', 'egl-sync', 'status', 'cleanup')
(15:13:47) INFO:    [Legendary]:       Updating game list
(15:13:47) INFO:    [Legendary]:       Game List Updated
(15:13:48) INFO:    [Legendary]:       Checking for game updates
(15:13:48) INFO:    [Legendary]:       Found 5 game(s) to update
(15:13:48) INFO:    [Gog]:             Found 0 game(s) to update
(15:13:51) INFO:    [Backend]:         Progress: Sugar /MiB/undefined
(15:13:51) INFO:    [Backend]:         Progress: Sugar /MiB/undefined
(15:13:51) INFO:    [Backend]:         checking for heroic updates
(15:13:52) INFO:    [Backend]:         finished updating
(15:13:54) INFO:    [Frontend]:        Refreshing Library
(15:13:54) INFO:    [Legendary]:       Refreshing library...
(15:13:54) INFO:    [Legendary]:       Updating game list
(15:13:54) INFO:    [Legendary]:       Game List Updated
(15:13:56) INFO:    [Legendary]:       Checking for game updates
(15:13:56) INFO:    [Legendary]:       Found 5 game(s) to update
(15:13:56) INFO:    [Gog]:             Found 0 game(s) to update
(15:14:40) INFO:    [Backend]:         Progress: Sugar /MiB/undefined
(15:14:40) INFO:    [Backend]:         Progress: Sugar /MiB/undefined
(15:14:42) INFO:    [Backend]:         finished updating
(15:14:44) INFO:    [Frontend]:        Refreshing Library
(15:14:44) INFO:    [Legendary]:       Refreshing library...
(15:14:44) INFO:    [Legendary]:       Updating game list
(15:14:44) INFO:    [Legendary]:       Game List Updated
(15:14:45) INFO:    [Legendary]:       Checking for game updates
(15:14:45) INFO:    [Legendary]:       Found 5 game(s) to update
(15:14:45) INFO:    [Gog]:             Found 0 game(s) to update
(15:14:54) INFO:    [Backend]:         Progress: Sugar /MiB/undefined
(15:14:54) INFO:    [Backend]:         Progress: Sugar /MiB/undefined
(15:14:55) INFO:    [Backend]:         finished updating
(15:14:57) INFO:    [Legendary]:       Getting More details with Legendary info command... 
(15:14:57) INFO:    [Frontend]:        Refreshing Library
(15:14:57) INFO:    [Legendary]:       Refreshing library...
(15:14:57) INFO:    [Legendary]:       Updating game list
(15:14:57) INFO:    [Legendary]:       Game List Updated
(15:14:59) INFO:    [Legendary]:       Checking for game updates
(15:14:59) INFO:    [Legendary]:       Found 5 game(s) to update
(15:14:59) INFO:    [Gog]:             Found 0 game(s) to update
(15:14:59) INFO:    [Legendary]:       Getting More details with Legendary info command... 
(15:15:07) INFO:    [Backend]:         Progress: Sugar /MiB/undefined
(15:15:07) INFO:    [Backend]:         Progress: Sugar /MiB/undefined
(15:15:08) INFO:    [Legendary]:       Getting More details with Legendary info command... 
(15:15:08) INFO:    [Backend]:         finished updating
(15:15:09) INFO:    [Legendary]:       Getting More details with Legendary info command... 
(15:15:09) INFO:    [Legendary]:       Getting More details with Legendary info command... 
(15:15:09) INFO:    [Legendary]:       Getting More details with Legendary info command... 
(15:15:10) INFO:    [Frontend]:        Refreshing Library
(15:15:10) INFO:    [Legendary]:       Refreshing library...
(15:15:10) INFO:    [Legendary]:       Updating game list
(15:15:10) INFO:    [Legendary]:       Game List Updated
(15:15:12) INFO:    [Legendary]:       Checking for game updates
(15:15:12) INFO:    [Legendary]:       Found 5 game(s) to update
(15:15:12) INFO:    [Gog]:             Found 0 game(s) to update
(15:16:00) INFO:    [Legendary]:       Getting More details with Legendary info command... 
(15:21:37) INFO:    [Legendary]:       Installed game list updated
(15:21:37) ERROR:   [Backend]:         SyntaxError: Unexpected end of JSON input
(15:21:37) INFO:    [Legendary]:       Installed game list updated
(15:21:37) ERROR:   [Backend]:         SyntaxError: Unexpected end of JSON input
(15:21:37) INFO:    [Legendary]:       Installed game list updated
(15:21:37) ERROR:   [Backend]:         SyntaxError: Unexpected end of JSON input
(15:21:37) INFO:    [Legendary]:       Installed game list updated
(15:21:37) ERROR:   [Backend]:         SyntaxError: Unexpected end of JSON input
(15:21:37) INFO:    [Legendary]:       Installed game list updated
(15:21:37) ERROR:   [Backend]:         SyntaxError: Unexpected end of JSON input
(15:21:37) INFO:    [Legendary]:       Installed game list updated
(15:21:37) ERROR:   [Backend]:         SyntaxError: Unexpected end of JSON input
(15:21:37) INFO:    [Legendary]:       Installed game list updated
(15:21:37) ERROR:   [Backend]:         SyntaxError: Unexpected end of JSON input
(15:21:37) INFO:    [Legendary]:       Installed game list updated
(15:21:37) ERROR:   [Backend]:         SyntaxError: Unexpected end of JSON input
(15:21:37) INFO:    [Legendary]:       Installed game list updated
(15:21:37) ERROR:   [Backend]:         SyntaxError: Unexpected end of JSON input
(15:21:37) INFO:    [Legendary]:       Installed game list updated
(15:21:37) ERROR:   [Backend]:         SyntaxError: Unexpected end of JSON input
(15:21:37) INFO:    [Legendary]:       Installed game list updated
(15:21:37) ERROR:   [Backend]:         SyntaxError: Unexpected end of JSON input
(15:21:37) INFO:    [Legendary]:       Installed game list updated
(15:21:37) ERROR:   [Backend]:         SyntaxError: Unexpected end of JSON input
(15:21:37) INFO:    [Legendary]:       Installed game list updated
(15:21:37) ERROR:   [Backend]:         SyntaxError: Unexpected end of JSON input
(15:21:37) INFO:    [Legendary]:       Installed game list updated
(15:21:37) ERROR:   [Backend]:         SyntaxError: Unexpected end of JSON input
(15:21:37) INFO:    [Legendary]:       Installed game list updated
(15:21:37) INFO:    [Legendary]:       Installed game list updated
(15:21:37) ERROR:   [Backend]:         SyntaxError: Unexpected end of JSON input
(15:21:37) INFO:    [Legendary]:       Installed game list updated
(15:21:37) ERROR:   [Backend]:         SyntaxError: Unexpected end of JSON input
(15:21:37) INFO:    [Legendary]:       Installed game list updated
(15:21:37) ERROR:   [Backend]:         SyntaxError: Unexpected end of JSON input
(15:21:37) INFO:    [Legendary]:       Installed game list updated
(15:21:37) ERROR:   [Backend]:         SyntaxError: Unexpected end of JSON input
(15:21:37) INFO:    [Legendary]:       Installed game list updated
(15:21:37) ERROR:   [Backend]:         SyntaxError: Unexpected end of JSON input
(15:21:37) INFO:    [Legendary]:       Installed game list updated
(15:21:37) ERROR:   [Backend]:         SyntaxError: Unexpected end of JSON input
(15:21:37) INFO:    [Legendary]:       Installed game list updated
(15:21:37) ERROR:   [Backend]:         SyntaxError: Unexpected end of JSON input
(15:21:37) INFO:    [Legendary]:       Installed game list updated
(15:21:37) ERROR:   [Backend]:         SyntaxError: Unexpected end of JSON input
(15:21:37) INFO:    [Legendary]:       Installed game list updated
(15:21:37) ERROR:   [Backend]:         SyntaxError: Unexpected end of JSON input
(15:21:37) INFO:    [Legendary]:       Installed game list updated
(15:21:37) ERROR:   [Backend]:         SyntaxError: Unexpected end of JSON input
(15:21:37) INFO:    [Legendary]:       Installed game list updated
(15:21:37) ERROR:   [Backend]:         SyntaxError: Unexpected end of JSON input
(15:21:37) INFO:    [Legendary]:       Installed game list updated
(15:21:37) ERROR:   [Backend]:         SyntaxError: Unexpected end of JSON input
(15:21:37) INFO:    [Legendary]:       Installed game list updated
(15:21:37) INFO:    [Legendary]:       Installed game list updated
(15:21:37) ERROR:   [Backend]:         SyntaxError: Unexpected end of JSON input
(15:21:37) INFO:    [Legendary]:       Installed game list updated
(15:21:37) INFO:    [Legendary]:       Installed game list updated
(15:21:37) INFO:    [Legendary]:       
(15:21:38) INFO:    [Frontend]:        Refreshing Library
(15:21:38) INFO:    [Legendary]:       Refreshing library...
(15:21:38) INFO:    [Legendary]:       Refreshing Epic Games...
(15:21:38) INFO:    [Legendary]:       Refreshing Epic Games...
(15:21:39) INFO:    [Legendary]:       usage: legendary [-h] [-v] [-y] [-V]
                 {auth,install,download,update,repair,uninstall,launch,list-games,list-installed,list-files,list-saves,download-saves,sync-saves,verify-game,import-game,egl-sync,status,cleanup}
                 ...
legendary: error: argument subparser_name: invalid choice: 'list' (choose from 'auth', 'install', 'download', 'update', 'repair', 'uninstall', 'launch', 'list-games', 'list-installed', 'list-files', 'list-saves', 'download-saves', 'sync-saves', 'verify-game', 'import-game', 'egl-sync', 'status', 'cleanup')
(15:21:39) INFO:    [Legendary]:       Updating game list
(15:21:39) INFO:    [Legendary]:       Game List Updated
(15:22:19) INFO:    [Legendary]:       
Checking for importable games...
The following games are importable (EGL -> Legendary):
 * Honeycreeper - Diabotical

Note: Only games that are also in Legendary's database are listed, if anything is missing run "list-games" first to update it.

Checking for exportable games...
Nothing to export.

Checking automatic sync...

(15:22:20) INFO:    [Frontend]:        Refreshing Library
(15:22:20) INFO:    [Legendary]:       Refreshing library...
(15:22:20) INFO:    [Legendary]:       Refreshing Epic Games...
(15:22:20) INFO:    [Legendary]:       Refreshing Epic Games...
(15:22:21) INFO:    [Legendary]:       usage: legendary [-h] [-v] [-y] [-V]
                 {auth,install,download,update,repair,uninstall,launch,list-games,list-installed,list-files,list-saves,download-saves,sync-saves,verify-game,import-game,egl-sync,status,cleanup}
                 ...
legendary: error: argument subparser_name: invalid choice: 'list' (choose from 'auth', 'install', 'download', 'update', 'repair', 'uninstall', 'launch', 'list-games', 'list-installed', 'list-files', 'list-saves', 'download-saves', 'sync-saves', 'verify-game', 'import-game', 'egl-sync', 'status', 'cleanup')
(15:22:21) INFO:    [Legendary]:       Updating game list
(15:22:21) INFO:    [Legendary]:       Game List Updated
(15:22:27) INFO:    [Backend]:         Progress: Sugar /MiB/undefined
(15:22:27) INFO:    [Backend]:         Progress: Sugar /MiB/undefined
(15:22:28) INFO:    [Backend]:         finished updating
(15:22:30) INFO:    [Frontend]:        Refreshing Library
(15:22:30) INFO:    [Legendary]:       Refreshing library...
(15:22:30) INFO:    [Legendary]:       Updating game list
(15:22:30) INFO:    [Legendary]:       Game List Updated
(15:22:31) INFO:    [Legendary]:       Checking for game updates
(15:22:31) INFO:    [Legendary]:       Found 5 game(s) to update
(15:22:31) INFO:    [Gog]:             Found 0 game(s) to update

**Comment by flavioislima** [comment]
Hmm, for some reason you are using a pretty outdated version of Legendary. It is like Heroic is trying to run legendary installed on your system.
Perhaps try to remove it first or update it to latest.

`Legendary Version: 0.20.6 A Red Letter Day`
Latest version is 0.20.25.

**Comment by wouter1984** [comment]
Perfect!! 

Thank you for the quick responce. I have updated Lendary to version 'Legendary v0.20.25" from github repo and now my library is updating again.

Keep up the good work with Heroic... Love that GOG is now also part of the launcher!

**Comment by happysmack** [comment]
I have 2 versions of Legendary installed apparently.
I have switched the binary to the newer version and am still getting the same results. And yes, I did restart.
If you notice that it states that it is still using the old version even though it is pointed to the newer versions path


![Screenshot from 2022-04-05 04-16-31](https://user-images.githubusercontent.com/10453370/161714261-9f3e6d7d-04bd-4c6d-91ee-5db6a2f22028.png)
![Screenshot from 2022-04-05 04-21-10](https://user-images.githubusercontent.com/10453370/161714276-d89de946-13f2-4225-8391-ff7fe380ff0e.png)
.
